imblearn/ensemble/tests/test_bagging.py

```diff
@@ -493,3 +494,12 @@ def test_max_samples_consistency():
         random_state=1)
     bagging.fit(X, y)
     assert bagging._max_samples == max_samples
+
+
+def test_bagging_multioutput_multilabel_error():
```

Comment:
Can we add those into the common tests instead?

Reply:
@glemaitre  do you mean as parametrized test in `imblearn.tests.test_common`?

Reply:
Partially yes. We need to add a test to `estimator_check.py`. No need to parametrize. Common tests are yielding the sampler one by one and give it to the tests to be used. Here we need to add a common test which will be used for the `classifier` basically.

Reply:
@glemaitre I think I am done

Reply:
No. I need to remove some code.

Reply:
@glemaitre what do you think?